### PR TITLE
Added support for detection of Yandex.Browser

### PR DESCRIPTION
**Pull request by beshkenadze**
Here is description: http://en.wikipedia.org/wiki/Yandex_Browser

### CODE REVIEW

File: bowser.js [hunk 1]
```diff
@@ -1,276 +1,290 @@
 /*!
-  * Bowser - a browser detector
-  * https://github.com/ded/bowser
-  * MIT License | (c) Dustin Diaz 2014
-  */
+ * Bowser - a browser detector
```
Comment (VovanR) [inline bowser.js]: Copyrights duplication

File: src/bowser.js [hunk 2]
```diff
@@ -1,270 +1,284 @@
+/*!
```
Comment (VovanR) [inline src/bowser.js]: Copyrights inserts from `src/copyright.js`

**Comment by beshkenadze** [comment]
Reset code formatting.

**Comment by ded** [comment]
:+1:

**Pull request merged**